### PR TITLE
feat: CSS tokenizer

### PR DESCRIPTION
**Pull request by bitpshr**
This pull request adds a new `css-parser` package that exposes a tolerant CSS tokenizer for the complete [CSS3 specification](https://www.w3.org/TR/css-syntax-3/#tokenization).

**TODO:**

1. Implement parser.
1. Add exhaustive tests.
1. Further verify spec compliancy.

### CODE REVIEW

File: packages/@romejs/css-parser/parse.test.ts [hunk 1]
```diff
@@ -0,0 +1,808 @@
+import "@romejs/core";
+import {test} from "rome";
+import {tokenizeCSS} from ".";
+
+test(
```
Comment (bitpshr) [inline packages/@romejs/css-parser/parse.test.ts:5]: These tests are brute-force and incomplete; they were really just a means of verifying output during implementation. I'd almost vote for a snapshot test of a very complex CSS file over manual tokenization checking like this, but it felt wrong not to include tests at all.

**Comment by sebmck** [comment]
Thank you for doing this, it looks great! I don't have any notes other than if any of the tests would value from using `t.inlineSnapshot` instead of `t.looksLike`?

**Comment by bitpshr** [comment]
@sebmck good call. Ideally I'd like to revisit the tests as a whole in a followup PR to fully hit each token's edge cases. I'll be sure to use `inlineSnapshot` to keep things tidy.

**Comment by sebmck** [comment]
Sure! I'll merge this and rebase #717 to make it use the new `ParserCore` methods I added.

**Comment by sebmck** [comment]
Thank you!
